webdriver/tests/websockets/connect.py

```diff
@@ -0,0 +1,43 @@
+import pytest
```

Comment:
I would suggest to put all the BiDi tests under a separate `bidi/` subfolder, and I think that this should also be done for the websocket tests.

Also the tests in this file are all around new sessions, so maybe a location at `webdriver/tests/bidi/new_session` would make sense?

Reply:
Thanks for the suggestion! This is done.